File: CSharp Pattern Programs/Symbol Patterns/Pattern 17.cs
using System;

class Pattern_97
 {
  public static void Main()
   {
    int n = 5; // size

    for (int i = 1; i <= n; i++)
     {
      for (int j = 1; j < n * 2; j++)
       {
        if (j <= i || j >= n * 2 - i)
         {
          Console.Write("*");
         }
        else
         {
          Console.Write(" ");
         }
       }

     Console.WriteLine();
     }
     Console.ReadKey(true);
   }
 }

File: JavaPatternPrograms/Symbol Patterns/Pattern 17.java
class Pattern_97
{
    public static void main(String args[])
    {
        int n = 5; // size

        for (int i = 1; i <= n; i++)
        {
            for (int j = 1; j < n * 2; j++)
            {
                if (j <= i  || j >= n * 2 - i)
                {
                    System.out.print("*");
                }
                else
                {
                    System.out.print(" ");
                }
            }

            System.out.println();
        }

    }
}